primitives/worker/src/block.rs

```diff
@@ -44,48 +42,50 @@ pub struct Block {
 	state_payload: Vec<u8>,
 }
 
-impl Block {
+impl BlockT for Block {
```

Comment:
I wondered whether I should move this definition to the sidechain/traits crate too. But I think for now it's fine.

Reply:
also extracted the struct definition